types/avro.go

```diff
@@ -12,10 +12,9 @@ import (
 )
 
 type AVROSchema struct {
-	Namespace string             `json:"namespace"`
```

Comment:
Remove the namespace field, this is not currently used except by the TableToAVRO function